nominal/core/client.py

```diff
@@ -78,6 +79,52 @@
 class NominalClient:
     _clients: ClientsBunch = field(repr=False)
 
+    @classmethod
+    def from_profile(
+        cls,
+        profile: str,
+        *,
+        trust_store_path: str | None = None,
+        connect_timeout: timedelta = timedelta(seconds=30),
+    ) -> Self:
+        """Create a connection to the Nominal platform from a named profile in the Nominal config.
+
+        Args:
+            profile: profile name in the Nominal config.
+            trust_store_path: path to a trust store certificate chain to initiate SSL connections. If not provided,
+                certifi's trust store is used.
+            connect_timeout: Request connection timeout.
+        """
+        config = NominalConfig.from_yaml()
+        prof = config.get_profile(profile)
+        return cls.from_token(
+            prof.base_url,
+            prof.token,
```

Comment:
This is a bit problematic, given that `from_token` expects the token first ;)

Reply:
I'm going to push a fix for this to the branch.

Reply:
good catch! I only decided to flip the function args in the signature later since it's annoying to retype the URL each time.